doc/client_setup.md

```diff
@@ -0,0 +1,199 @@
+## Setting up a new C2 client
+
+This document describes the details of setting up a new C2 client.
+
+### Overview
+
+A *client* represents a group of Users within the C2 system that share
+a data model. Existing examples include **GSA18F** and **NCR**.
+
+Within a *client* users may be assigned different roles. See the [Authorization](roles.md)
+document for details on roles.
+
+A *client* namespace is assigned to each User via a `client_slug` model value. The
+convention is that the `client_slug` is a downcased string matching the client model
+namespace. For example `ncr` for **NCR**.
+
+A User without an assigned `client_slug` may not create, edit, view or search Proposals,
```

Comment:
same goes for users without a client slug at all. They won't see a link to create a work order or procurement, but it is possible to do so.